Tella/Domain/Entity/GDrive/GDriveServer.swift

```diff
@@ -17,31 +17,21 @@ class GDriveServer: Server {
 
     init(id: Int? = nil,
          name: String? = "Google Drive",
-         serverURL: String? = nil,
-         username: String? = nil,
-         password: String? = nil,
-         accessToken: String? = nil,
          rootFolder: String,
-         serverType: ServerConnectionType? = .gDrive,
-         allowMultiple: Bool? = false
-    ) {
+         serverType: ServerConnectionType? = .gDrive) {
+        
         self.rootFolder = rootFolder
         super.init(id: id,
                    name: name,
-                   serverURL: serverURL,
-                   username: username,
-                   password: password,
-                   accessToken: accessToken,
-                   serverType: serverType
-        )
+                   serverType: serverType,
+                   allowMultipleConnections: false)
     }
 
     required init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         self.rootFolder = try container.decode(String?.self, forKey: .rootFolder)
         try super.init(from: decoder)
         self.serverType = .gDrive
-        self.allowMultiple = false
```

Comment:
We need to set allowMultiple as false here, otherwise, it will appear as nil when fetching the servers from the db. this is because the Server class does not handle the decoding of allowMultiple, since is not on the db we are doing this dynamically according to the type of the connection.

Reply:
yeah you're right